packages/flutter_tools/lib/src/commands/upgrade.dart

```diff
@@ -97,12 +105,19 @@ class UpgradeCommandRunner {
     @required GitTagVersion gitTagVersion,
     @required FlutterVersion flutterVersion,
     @required bool testFlow,
+    @required bool verifyOnly,
   }) async {
     final String upstreamRevision = await fetchRemoteRevision();
     if (flutterVersion.frameworkRevision == upstreamRevision) {
       globals.printStatus('Flutter is already up to date on channel ${flutterVersion.channel}');
       globals.printStatus('$flutterVersion');
       return;
+    } else if(verifyOnly) {
+      globals.printStatus('A new version of Flutter is available!\n');
```

Comment:
Can you update this to:

```dart
      globals.printStatus('A new version of Flutter is available on channel ${flutterVersion.channel}\n');
// TODO(fujino): use a [FlutterVersion] once that class supports arbitrary revisions.
      globals.printStatus('The latest revision: $upstreamRevision\n', emphasis: true);
      globals.printStatus('Your current version: ${flutterVersion.frameworkRevision}');
```

Reply:
Looks like you'll have to update test expectation if you make this change.

Reply:
>Can you update this to:

Sure, will do that!

>Looks like you'll have to update test expectations if you make this change (sorry).

I'm not quite sure that I follow.

Reply:
Oh I get that now, I would have to change the tests, my bad.

Reply:
No problem! I just wanted to save you the time in case you missed it, pushed it up to GitHub, and found out 30 minutes later that CI is failing. Because that's basically the story of my life :)